Src/stm32f4xx_it.c

```diff
@@ -1014,11 +1035,13 @@ void GCN64_CommandStart(uint8_t player)
 				}
 				else
 				{
-					serial_interface_output((uint8_t*)"A", 1);
+					if(frame == NULL) // there was a buffer underflow
+						serial_interface_output((uint8_t*)"A\xB2", 2);

```

Comment:
Couldn't frame also == NULL if we're waiting for rumble? It's not necessarily a buffer underflow I think if the code reaches this part of the workflow.

Reply:
Fixed in latest commit.